core/src/main/web/app/helpers/katexhelper.js

```diff
@@ -0,0 +1,194 @@
+/* eslint no-console:0 */
```

Comment:
add this to top of the file:
```
copied from https://github.com/Khan/KaTeX/tree/master/contrib/auto-render
until https://github.com/Khan/KaTeX/issues/425 is resolved.
```
then in comment include their license:
https://github.com/Khan/KaTeX/blob/master/LICENSE.txt